doc/howto/gh-actions-badges.md

```diff
@@ -0,0 +1,84 @@
+# Use GitHub Actions To Place A MyBinder.org Badge On Pull Requests
+
+You can facilitate review of GitHub pull requests on [mybinder.org](https://mybinder.org) by using [GitHub Actions](https://github.com/features/actions) to place a ![Binder](https://mybinder.org/badge_logo.svg) badge or link on Pull Requests.  Below are several examples of how to accomplish this.
+
+To enable GitHub Actions, you must place .yaml files that define your GitHub Action workflow in the `/.github/workflows/` directory in your repository.
+
+## Example 1: Comment on pull request with a binder badge
+
+_Note: this example will not work on pull requests from forks.  This is to protect repositories from malicious actors.  To trigger a GitHub Action with sufficient permissions to comment on a pull request from a fork, you must trigger the action via another event, such as a comment or a label.  This is demonstrated in Example 2 below._
```

Comment:
can you make this a Sphinx note so it renders nicely? Use the structure:

````
```{note}
content
```
````

Reply:
done